src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -33,25 +40,115 @@ exports.handler = async function (event, context, callback) {
         message: `Alert for ${data.id} - TEST `,
       });
     } else {
-      const { emailRecipients, sourceEmail } = await getSecretsValue({
+      let isProgramTypeChp = false;
+      if (data.programType == "CHP") {
+        secretId = `${project}/${stage}/alerts/CHP`;
+        isProgramTypeChp = true;
+      }
+      //{ emailRecipients, sourceEmail, emailRecipientsA, emailRecipientsB }
+      const {
+        emailRecipients,
+        sourceEmail,
+        emailRecipientsA,
+        emailRecipientsB,
+      } = await getSecretsValue({
         region,
         secretId,
       });
+      let recipientType;
+      let recipients;
+      // if it greater then 2 days but less then 4 days
+      if (
+        data.secSinceMmdlSigned > 48 * 3600 &&
+        data.secSinceMmdlSigned < 48 * 2 * 3600
+      ) {
+        recipientType = "emailRecipientsA";
+        recipients = emailRecipientsA;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned > (48 * 3600)) && (data.secSinceMmdlSigned < ((48 * 2) * 3600))":
+            data.secSinceMmdlSigned > 48 * 3600 &&
+            data.secSinceMmdlSigned < 48 * 2 * 3600,
+        });
```

Comment:
all these console tables really seem to clutter things up, do we really need them or are they just for debugging?